src/config.ts

```diff
@@ -94,6 +98,16 @@ export function getAppConfig({
   }
 }
 
+function calculateMinTTL(POSTAGE_TTL_MIN: string | undefined) {
+  if (Number(POSTAGE_TTL_MIN) >= DEFAULT_POSTAGE_TTL_MINIMUM) {
+    return POSTAGE_TTL_MIN
+  }
+  logger.warn(`To extend postage stamps, POSTAGE_TTL_MIN needs to be at least ${DEFAULT_POSTAGE_TTL_MINIMUM} seconds.`)
+  logger.warn(`ttlMin setting is being increase to this value automatically.`)
```

Comment:
What I like about this approach is that we are automating the basic information we need for the feature to work. It's something new on the whole procedure, that might be the reason it looks off. The other way around it is to make it a mandatory setting and to be above 60 seconds

Reply:
Just to expand a bit on it. My problem is not with having "automation" like default values. My problem is with the logic that if the user specifies (either by accident or intentionally) a value below the threshold then we mutate it to some other value. Because then he will get different behavior than he expected. And while yes, you issue warning in logs, in my experience not many people read logs until there is something wrong. Which for example in this case could be even hard to detect.

Reply:
Yes, that was exactly my idea. Until I washed my hands assuming the users were going to read the log warnings when the script start. If we remove the assumption out of the formula, we end up with a mandatory field with a specific value above 60 seconds, which is the solution I came up with on the last pushed code